overlord/snapshotstate/snapshotmgr.go

```diff
@@ -304,5 +383,23 @@ func doForget(task *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("internal error: task %s (%s) snapshot info is missing the filename", task.ID(), task.Kind())
 	}
 
+	// in case it's an automatic snapshot, remove the set also from the state (automatic snapshots have just one snap per set).
+	if rmerr := removeSnapshotState(st, snapshot.SetID); rmerr != nil {
```

Comment:
s/rmerr/err/ , used this way in the if it's scoped to it

Reply:
Done